<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widgetConfiguration</class>
 <widget class="QWidget" name="widgetConfiguration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>216</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>3</y>
     <width>301</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::ExpandingFieldsGrow</enum>
    </property>
    <item row="1" column="0">
     <widget class="QLabel" name="lblDrive">
      <property name="text">
       <string>Installation Drive</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QSlider" name="sizeSlider">
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QProgressBar" name="pbFreeSpace">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <widget class="Line" name="hLine">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="lblUsername">
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="txtUsername"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="lblPassword">
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="txtPassword">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="LblRetypePassword">
      <property name="text">
       <string>Retype Password </string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="txtRetypePassword">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="lblFree">
      <property name="text">
       <string>After Installation</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="lblInstallation">
      <property name="text">
       <string>Installation Size</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="comboDrive"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="lblFreeLeft">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>70</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>GB free</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblSize">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string> GB</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblDriveFreeSpace">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>GB free</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
